onnxruntime/python/tools/bert/bert_model_optimization.py

```diff
@@ -1295,6 +27,12 @@ def main():
     parser.add_argument('--float16', required=False, action='store_true')
     parser.set_defaults(float16=False)
 
+    parser.add_argument('--gpu_only', required=False, action='store_true')
+    parser.set_defaults(gpu_only=False)
+
+    parser.add_argument('--tensor_flow', required=False, action='store_true')
```

Comment:
How about --model_type with enum like "pytorch", "tensorflow" etc.
It is because we might need to support more model types in the future.